tests/pdf_viewer/test_pdf_download.py

```diff
@@ -0,0 +1,60 @@
+import os
+import platform
+import time
+
+import pytest
+from pynput.keyboard import Controller, Key
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.browser_object_navigation import Navigation
+from modules.page_object import GenericPdf
+
+
+@pytest.fixture()
+def add_prefs():
+    return []
+
+
```

Comment:
You'll need to add a pytest mark called `headed` for this to run in CI, as you're using pynput. You could also theoretically try setting the `browser.download.manager.showWhenStarting` to `False` to avoid the dialog.

Reply:
I searched a lot for this preference in order to avoid the multiple sending of Enter and Tab keys in order to make it work on all OS, but could not find it yet.